### DISCUSSION TITLE
yaml: line 44: did not find expected key

### DISCUSSION THREAD

**Discussion by Mr-G-Ncongwane** [outline/outline#7359] [General]
I'm trying to create an Outline using Docker and Ubuntu, I copied the Docker-compose.yml and docker.env as it is from the [Docker Outline documentation ](https://docs.getoutline.com/s/hosting/doc/docker-7pfeLP5a8t), when I run `docker compose run --rm outline yarn db:create --env=production-ssl-disabled` command I get an error saying "yaml: line 44: did not find expected key".

![image](https://github.com/user-attachments/assets/72638c7b-e040-4f21-8f40-f42ba74ea0ea)

Any idea how can I resolve this?. Thank you

**Comment by tommoor** [comment]
Your YAML file is invalid as the error states, but as you didn't post the configuration I don't think anyone can help further here.

**Reply by Mr-G-Ncongwane** [reply]
[docker.txt.txt](https://github.com/user-attachments/files/16520711/docker.txt.txt)
[docker-compose.txt](https://github.com/user-attachments/files/16520714/docker-compose.txt)

Hi @tommoor  here are my files, I am trying to create a basic Outline running on docker container. Both files are located under /home/eduvos/Outline

**Reply by tommoor** [reply]
The yaml is valid, so I'm not sure where that error would be coming from

**Reply by Mr-G-Ncongwane** [reply]
I'm stuck, perhaps I should try with docker desktop
